src/brogue/Items.c

```diff
@@ -2354,6 +2356,14 @@ void itemDetails(char *buf, item *theItem) {
                 strcat(buf, buf2);
             }
 
+            if (theItem->lastUsed > 0) {
+                turnsSinceLastUse = rogue.playerTurnNumber - theItem->lastUsed;
+                sprintf(buf2, "You last used it %i turn%s ago. ",
```

Comment:
Should the number be given a color?

Reply:
no, there's not really a precedent for that currently